updater/updater/updater.go

```diff
@@ -14,6 +15,7 @@ type AdminConfig struct {
 	URI      string
 	Username string
 	Password string
+	MTLS     *tls.Config
```

Comment:
I would prefer to limit the amount of configuration data passed to the library and only pass `ClientCertificates []tls.Certificate` instead of `MTLS *tls.Config`.